### DISCUSSION TITLE
top 3 from pathways="auto" different from top common pathways from OncogenicPathways()

### DISCUSSION THREAD

**Discussion by AnitaS321** [PoisonAlien/maftools#758] [Q&A]
Hello,
I would like include on my oncoplot the top 3 pathways I see when using OncogenicPathways function. I cannot figure out why the reported Fraction_mutated_samples is different depending on the function used as the maf is the same. Or, is it choosing from the top 3 based on the genes shown in the oncoplot? Is there a way to simply choose the names of the pathways I want to include from OncogenicPathways() function, without having to create a tsv file that lists each gene for my chosen pathways? I have removeNonMutated = FALSE so that all samples should be considered. Thanks in advance for any help!
<img width="1109" alt="Screen Shot 2021-08-04 at 4 39 50 PM" src="https://user-images.githubusercontent.com/26095450/128251810-dd06f969-397a-4df9-8874-4d4bb0d861c8.png">

<img width="537" alt="Screen Shot 2021-08-04 at 4 15 53 PM" src="https://user-images.githubusercontent.com/26095450/128251731-d6698675-d60f-4776-8f3c-2bbc1a55c477.png">

**Comment by PoisonAlien** [comment]
Hi,

Sorry for the tardy response. Just got back to work.
Just to confirm, `oncoplot` and `OncogenicPathways` both use different sources for pathways. 

1. `oncoplot` uses pathways from derived from driver genes. [Source](https://www.cell.com/action/showPdf?pii=S0092-8674%2818%2930237-X) 
2. `OncogenicPathways` uses pathways from the TCGA [publication](https://www.cell.com/cell/pdf/S0092-8674(18)30359-3.pdf)

Hope this helps.

**Reply by AnitaS321** [reply]
No worries - thank you so much for explaining!
